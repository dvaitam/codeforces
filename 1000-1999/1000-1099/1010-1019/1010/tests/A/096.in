7
1
3 7 10 8 10 2 7
10 3 2 5 7 3 9
