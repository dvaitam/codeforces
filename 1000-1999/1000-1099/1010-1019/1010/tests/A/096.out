46.2608024691
